x-pack/test/functional/apps/ml/short_tests/model_management/model_list.ts

```diff
@@ -144,6 +156,26 @@ export default function ({ getService }: FtrProviderContext) {
           { pipelineName: `pipeline_${modelWithPipelineData.modelId}`, expectDefinition: false },
         ]);
       });
+
+      describe('Add trained model flyout', () => {
+        it('should display elements on Manual Download tab correctly', async () => {
+          await ml.testExecution.logTestStep('Open the Add Trained Model Flyout');
+          await maybeClearSearchInput();
+          await ml.trainedModelsFlyout.assertFlyoutOpen();
```

Comment:
I believe the assertion method should not have side effects, especially invoking an action it asserts. Instead, we can create an `open` method that clicks the button to open it, and the asserts it is open.

Reply:
I like that idea.  I'll jump on that pronto.